specification/components/schemas/extensions/Extension-eRS-ReferralState.yaml

```diff
@@ -0,0 +1,114 @@
+      | `state`                                    | `1..1`      | Outlines the current state of the `ReferralRequest`                              |
+      | `reason`                                   | `0..1`      | Details the reason why the `ReferralRequest` is in this state, where appropriate |
+      | `comments`                                 | `0..1`      | Any comments around the current `ReferralRequest` state                          |
+    items:
+      anyOf:
+        - type: object
+          required:
+            - url
+            - valueCoding
+          properties:
+            url:
+              type: string
+              enum:
+                - 'state'
+              example: 'state'
+            valueCoding:
+              type: object
+              required:
+                - system
+                - code
+                - display
+              properties:
+                system:
+                  type: string
+                  enum:
+                    - 'https://fhir.nhs.uk/STU3/CodeSystem/eRS-ReferralState-1'
+                  example: https://fhir.nhs.uk/STU3/CodeSystem/eRS-ReferralState-1
+                code:
+                  type: string
+                  enum:
+                    - REJECTED
+                    - CANCELLED_REFERRAL
+                    - DID_NOT_ATTEND
+                    - ASSESSMENT_RESULT
+                    - TRIAGE_RESPONSE
+                    - APPT_CANCELLED_BY_PROVIDER
+                    - APPT_CANCELLED_BY_PATIENT
+                    - AWAITING_TRIAGE
+                    - NOT_BOOKED
+                    - DEFERRED_TO_PROVIDER
+                    - BOOKED
+                    - TRIAGED_PROVIDER_TO_ACTION
+                  example: 'APPT_CANCELLED_BY_PATIENT'
+                display:
+                  type: string
+                  enum:
+                    - 'Rejected'
+                    - 'Cancelled Referral'
+                    - 'Did Not Attend'
+                    - 'Assessment Result'
+                    - 'Triaged Response'
+                    - 'Appointment Cancelled By Provider'
+                    - 'Appointment Cancelled By Patient'
+                    - 'Awaiting Triage'
+                    - 'Not Booked'
+                    - 'Defer to Provider'
+                    - 'Booked'
+                    - 'Triaged - Provided To Action'
+                  example: 'Appointment Cancelled By Patient'
+        - type: object
+          required:
+            - url
+            - valueCoding
+          properties:
+            url:
+              type: string
+              enum:
+                - 'reason'
+              example: 'reason'
+            valueCoding:
+              type: object
+              required:
+                - system
+                - code
+              properties:
+                system:
+                  type: string
+                  example: 'https://fhir.nhs.uk/STU3/CodeSystem/eRS-ReviewOutcome-1'
```

Comment:
Was there a way to include multiple examples? It would be nice if we could include the other code systems here.